### PR TITLE
Create WORKFLOW.md

### PR DESCRIPTION
**Pull request by RichiH**
The release section needs to be updated with your preferences, else this should be good

### CODE REVIEW

File: WORKFLOW.md
Comment (RichiH) [reply WORKFLOW.md:30]: team maintainer would be wrong the others are distinct

File: WORKFLOW.md
Comment (RichiH) [reply WORKFLOW.md:49]: Yes, don't carry PRs for ages, keep short cycles. Less of a concern in Tempo today, but quite a substantial one in Grafana, and we want to keep the same doc for all of the projects.

**Review by joe-elliott — APPROVED** [review]

**Comment by joe-elliott** [comment]
Approved, but leaving open for a bit if a community member wants to comment.

**Pull request merged**
